BeerCatalogFullstack/DataAccess/Repositories/BeerRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataAccess.Core;
+using DataAccess.Models;
+
+namespace DataAccess.Repositories
+{
+    public class BeerRepository : GenericRepository<Beer>
+    {
+        public BeerRepository(ApplicationContext context) : base(context) { }
+
+        public IReadOnlyList<Beer> GetAll()
+        {
+            return Get()
+                .ToList();
+        }
```

Comment:
You have to remove unnecessary methods when you update the codebase. Do not forget about it.